src/Form/Type/TaxonAttributeValueType.php

```diff
@@ -26,7 +26,7 @@ class TaxonAttributeValueType extends AbstractResourceType
     public function __construct(
         #[Autowire(TaxonAttributeValue::class)]
         string $dataClass,
-        #[Autowire('sylius_taxon_attribute_value.validation_groups')]
+        #[Autowire('%sylius.form.type.product_attribute_value.validation_groups%')]
```

Comment:
I think you can't use the **Product**AttributeValue validation group for the **Taxon**AttributeValue

Reply:
i looked good when i tested it back then :face_with_diagonal_mouth:

Reply:
it looked good when i tested it back then 🫤

Reply:
Yeah, but if someone modify that validation group for product_attribute_value, it will change for this too. 
Maybe when we declare a resource, sylius create automatically a new parameter for validation_groups ? 

We should check if parameter `sylius.form.type.taxon_attribute_value.validation_groups` exist, or create it otherwise
Maybe it's name is related to the resource name, so maybe `synolia_akeneo` instead `sylius`.